web/pdf_viewer.js

```diff
@@ -565,7 +565,7 @@ var PDFViewer = (function pdfViewer() {
         pageView.viewport.convertToViewportPoint(x + width, y + height)
       ];
       var left = Math.min(boundingRect[0][0], boundingRect[1][0]);
-      var top = Math.min(boundingRect[0][1], boundingRect[1][1]);
+      var top = Math.max(Math.min(boundingRect[0][1], boundingRect[1][1]), 0);
```

Comment:
Would it perhaps be a good idea to add a comment here, explaining why is necessary?